src/mobile/apps/feature/index.coffee

```diff
@@ -14,11 +14,12 @@ app.set 'view engine', 'jade'
 app.get '/feature/:id', routes.index
 app.get '/feature/:id/:queryParams', routes.index
 
+# TODO: make sure we remove this route
```

Comment:
Maybe instead of these TODOs we can create a ticket in Jira, within the project epic, linking to these two lines to remove after release?

Another option is to make these TODOs more specific by linking to the appropriate Jira epic.

Reply:
+1 commented similar in Slack. Also, if this ticket is about wiring up bid2, is it possible to that in this PR? Would it be analogous to the diff already present?

Reply:
> Also, if this ticket is about wiring up bid2, is it possible to that in this PR?

The current diff should already accomplish that. 

The `auction_reaction/server.tsx` addition should match the request URL and proxy down to the route defined in Reaction: https://github.com/artsy/reaction/blob/master/src/Apps/Auction/routes.tsx#L19

Reply:
Because we match the `reaction_server` route first? Thank you for clarifying!

Reply:
Yep. We'll match on Force's Express route first then the app defined in artsy/reaction will match on the [Found](https://github.com/4Catalyzer/found) (a React router) route.

Reply:
I have added this as AC in 🔒 [AUCT-465](https://artsyproduct.atlassian.net/browse/AUCT-465)
> - Audit and remove unused desktop AND mobile routes when setting up the /bid route as part of release (because we are currently using /bid2 this doesn’t yet affect mobile routes)

Reply:
Actually I am going to create a whole new ticket for this but the point stands.